internal/cli/up.go

```diff
@@ -62,12 +62,13 @@ func (c *upCmd) run(args []string) error {
 		return err
 	}
 
-	req := proto.CreateServiceRequest{Service: service, Watch: c.watch, LogLevel: proto.LogLevel(logLevel())}
-
-	err = dCli.CreateService(ctx, req)
-	s, ok := status.FromError(err)
-	if ok && s.Code() == codes.Unknown {
-		return errors.New(s.Message())
+	for i := range service {
```

Comment:
maybe rename this to `services` since it now contains many services, not just one?